app/src/main/java/net/gsantner/opoc/frontend/filebrowser/GsFileBrowserOptions.java

```diff
@@ -119,6 +121,8 @@ public static class Options {
 
         public Collection<File> favouriteFiles, recentFiles, popularFiles = null;
         public GsCallback.a1<CharSequence> setTitle = null, setSubtitle = null;
+
+        public GsCallback.a0 refresh = null;
```

Comment:
Reverted back to just having collections and added a refresh callback here. The refresh callback is called in `loadFolder()` and can be used to do whatever.